src/app/components/ScrollLock.tsx

```diff
@@ -21,6 +21,10 @@ const ScrollLockBodyClass = createGlobalStyle`
   }
 
   @media print {
+    #root {
+      display: none;
```

Comment:
this will hide content when printing unconditionally because of the toc overlay, you need to put it in the conditionals on lines 14 and 19